app/encryption.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+import pgpy
+
+
+class EncryptionFailedException(Exception):
+    pass
+
+
+def pgp_encrypt_message(message):
+    # A key can be loaded from a file, like so:
+    our_key, _ = pgpy.PGPKey.from_file(os.getenv('OUR_PUBLIC_KEY_PATH'))
+    other_key, _ = pgpy.PGPKey.from_file(os.getenv('OTHER_PUBLIC_KEY_PATH'))
```

Comment:
We have multiple different supplier keys that need to be chosen based on pack code

Reply:
we didn't cater for that requirement because we knew you'd been working on a branch late last Wednesday and thought you'd probably have a good solution figured out for how to map pack codes to keys and directories